### DISCUSSION TITLE
Transit -> Transient?

### DISCUSSION THREAD

**Discussion by jdtsmith** [kickingvegas/casual-suite#9] [General]
Thanks for rolling these together.  It would be great to better emphasize the role of transient in enabling casual-suite.  The acknowledgments all mention "Transit"; perhaps that was supposed to be "transient"?   Also would be good to link to the transient docs for people who wonder how to make their own "casual" things.

**Comment by kickingvegas** [comment]
Oh derp. Thanks for catching this typo. Will amend.

**Comment by kickingvegas** [comment]
Fixed for all Casual porcelains. https://melpa.org/#/?q=kickingvegas

Closing.
